Cannot find module 'escope'

When I try to use eslint with this config I get this error message:
```
Error: Cannot find module 'escope'
Require stack:
- /usr/local/lib/node_modules/eslint/lib/api.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at getModules (/home/zsolt/Documents/zsolt.mate/javascript-academy/node_modules/eslint-config-client-shiwaforce/node_modules/babel-eslint/index.js:36:25)
    at Object.exports.parse (/home/zsolt/Documents/zsolt.mate/javascript-academy/node_modules/eslint-config-client-shiwaforce/node_modules/babel-eslint/index.js:368:19)
    at parse (/usr/local/lib/node_modules/eslint/lib/linter/linter.js:806:29)
    at Linter._verifyWithoutProcessors (/usr/local/lib/node_modules/eslint/lib/linter/linter.js:1344:33)
    at Linter._verifyWithConfigArray (/usr/local/lib/node_modules/eslint/lib/linter/linter.js:1729:21)
    at Linter.verify (/usr/local/lib/node_modules/eslint/lib/linter/linter.js:1471:65)
    at Linter.verifyAndFix (/usr/local/lib/node_modules/eslint/lib/linter/linter.js:1953:29)
```
The package is using 
babel-eslint 7.2.3
ESlint 8.12.0

--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
     "url": "https://github.com/shiwaforce/eslint-config-client-shiwaforce/issues"
   },
   "dependencies": {
-    "babel-eslint": "^7.1.1",
+    "babel-eslint": "10.1.0",
     "extend": "^3.0.0"
   },
   "description": "Shareable client side config for eslint by ShiwaForce",

